modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2720,6 +2720,74 @@ definition {
 		}
 	}
 
+	@description = "Verify that the Is Not Equals to operator is available when adding filters for the integer fields to select data from the object"
+	@priority = "4"
+	test IsEqualToIsAvailableForIntegerFields {
+		task ("Given: A custom object with an integer field type") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject",
+				objectName_2 = "CustomObject",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Integer",
+				fieldName = "customInteger",
+				fieldType = "Integer");
+
+			ObjectAdmin.addObjectFieldViaUI(
+				field = "Custom Integer",
+				fieldLabel = "Custom Aggregation",
+				fieldType = "Aggregation",
+				function = "Max",
+				relationship = "Relationship");
+		}
+
+		task ("When: A new filter is created with the custom integer field with Is Equal To selected") {
+			ObjectAdmin.goToFieldsDetails(label = "Custom Aggregation");
+
+			Click(locator1 = "ObjectCustomViews#ADD_BUTTON_FILTERS");
+
+			Click(locator1 = "ObjectCustomViews#VIEW_MODAL_CONTENT");
+
+			Click(
+				key_picklistOption = "Custom Integer",
+				locator1 = "ObjectCustomViews#PICKLIST_FILTER_KEBAB_MENU_OPTION");
+
+			Click(
+				key_filterLabel = "Filter Type",
+				locator1 = "ObjectCustomViews#SELECT_FILTER_GENERIC");
+
+			Click(
+				key_fieldType = "Is Equal To",
+				locator1 = "ObjectField#TYPE_OPTION");
+
+			Type(
+				locator1 = "ObjectCustomViews#AGGREGATION_FILTER_VALUE",
```

Comment:
TextInput#ANY is fine here

Reply:
TextInput#ANY doesn't work for me on this value field

Reply:
With "Value" as the variable? I remember trying it and working but maybe I'm misremembering. I do remember "value" without the capital "V" doesn't work.

Reply:
I think I did try out with 'value' at first, but 'Value' works for me thank you! I'll change this and remove the path
`			Type(
				key_text = "Value",
				locator1 = "TextInput#ANY",
				value1 = "1");`